code/game/mecha/mech_bay.dm

```diff
@@ -22,6 +25,22 @@
 	recharging_mecha = null
 	return ..()
 
+
+/turf/simulated/floor/mech_bay_recharge_floor/attackby(obj/item/C as obj, mob/user as mob, params)
+	if(!C || !user) // I don't know what these do, they're leftovers from floor.dm
+		return 1
+	if(..()) // Ditto
+		return 1
+	if(intact && istype(C, /obj/item/weapon/screwdriver)) //Can't be a crowbar, because otherwise the attackby() for floor.dm intercepts the call, I think
+		to_chat(user, "<span class='danger'>You disassemble the mech bay recharge station.</span>")
+		new /obj/item/stack/sheet/plasteel(src, 2)
+		qdel() //calls the proc above that stops the machine from working - Not required with BeforeChange, but that isn't working.

```

Comment:
Wait what

<a href=https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/garbage_collection/garbage_collector.dm#L91-L93>I'm not sure qdel does anything without an argument, bucko</a>